ioXt/uraniborg/AndroidStudioProject/Hubble/app/src/main/java/com/uraniborg/hubble/MainActivity.java

```diff
@@ -65,6 +70,9 @@ private boolean initialize() {
     mHardwareInfo = new HardwareInfo();
     mBuildInfo = new BuildInfo();
     mDeviceProps = new DevicePropertiesInfo();
+    mHandlerThread = new HandlerThread("HubbleWorkerThread");
+    mHandlerThread.start();
+    mHandler = new Handler(mHandlerThread.getLooper());
```

Comment:
Could you change this to use `Executors#newSingleThreadExecutor()` (or similar) instead of a `HandlerThread`?

Reply:
Updated and good to know `Executors#newSingleThreadExecutor()` since I don't write Android app for a while. :)